.storybook/blocks/ComponentHeader/ComponentHeader.tsx

```diff
@@ -33,7 +33,7 @@ export const ComponentHeader: React.FC<ComponentHeaderProps> = ({
     <div className={styles.component}>
```

Comment:
I see `children` and `npmPackage` two unused props in this component and I feel they could be removed. Though it is not in scope of this PR so I don't feel strongly about this comment, you can use your judgement and decide on this. 
And I would suggest using default value for stage as Oleks suggested.